### ISSUE TITLE
Graylog2 pipeline rules: Undo without undo history erases rule

### ISSUE THREAD

**Issue by bronius** [Graylog2/graylog-plugin-pipeline-processor#224]
### Problem description
Hitting Undo (cmd/ctrl+z) in the text editor on the "Rule source" field when there is no more Undo History erases the entire text field. Workaround is to Redo (cmd/ctrl+shift+z).

### Steps to reproduce the problem

1. Edit an existing Rule in Pipeline
2. With cursor in Rule source text area, hit Undo
3. See that the text field is cleared out and a red X indicating syntax error appears

### Environment

* Graylog Version: Graylog 2.3.1+9f2c6ef
* Pipeline Processor plugin version:
* Elasticsearch Version:
* MongoDB Version:
* Operating System:
* Browser version:

**Comment by joschi** [comment]
@bronius I'm pretty sure that's something your browser is doing and not the Graylog web interface.

Which web browser and which operating system (exact version) are you using?

**Comment by bronius** [comment]
I don't think so, because this is not the behavior in a standard text field, and I've not seen it on other text fields (which of course RTF field UI developers have to wrap with javascript to make the magic happen).  Indeed I just checked here on Github by clicking Edit on my original post above, and if it were an overall browser/env issue, one might expect it to occur here as well, and it did not.

Mac El Capitan 10.11.6 / Chrome Version 61.0.3163.100 (Official Build) (64-bit)

I could try another browser/OS some time, but in the meantime, I just wanted to call it to someone's attention that this is the behavior I observed.
